tasks/__init__.py

```diff
@@ -12,6 +12,7 @@
 from .manifest import manifest
 from .upgrade import upgrade
 from .test import test
+from .changelog import update_changelog
```

Comment:
Do we want to add it to the `root` like the other tasks ?

Reply:
Uhm it seems tasks are automatically picked up at import time, will make this consistent

Reply:
Strange, when I wrote a task for the `datadog-agent`ci, I couldn't invoke the task before adding it to the root.